data/data/bootstrap/files/usr/local/bin/installer-gather.sh

```diff
@@ -0,0 +1,104 @@
+#!/usr/bin/env bash
+set -eo pipefail
+
+ARTIFACTS="${1:-/tmp/artifacts}"
+
+echo "Gathering bootstrap journals ..."
+mkdir -p "${ARTIFACTS}/bootstrap/journals"
+for service in bootkube openshift kubelet crio
+do
+    journalctl --boot --no-pager --output=short --unit="${service}" > "${ARTIFACTS}/bootstrap/journals/${service}.log"
+done
+
+echo "Gathering bootstrap containers ..."
+mkdir -p "${ARTIFACTS}/bootstrap/containers"
+sudo crictl ps --all --quiet | while read -r container
```

Comment:
does `crictl ps --all --quiet` include containers which have terminated? (e.g. due to invalid flags) ?

Reply:
Yeah, it should list these too - `--all` includes all states